.eslintrc.js

```diff
@@ -1,24 +1,29 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
-const { getBaseConfig } = require('@edx/frontend-build');
+const { createConfig } = require('@edx/frontend-build');
 
-const config = getBaseConfig('eslint');
-
-config.overrides = [
-  {
-    files: ['*.test.js', '*.test.jsx'],
-    rules: {
-      'react/prop-types': 'off',
-      'react/jsx-no-constructed-context-values': 'off',
+const config = createConfig('eslint', {
+  extends: [
+    '@edx/eslint-config',
+    'plugin:@tanstack/eslint-plugin-query/recommended',
```

Comment:
[note] Extending this plugin is the functional change here: https://tanstack.com/query/v4/docs/react/eslint/eslint-plugin-query. In the future, this plugin may end up in `@edx/frontend-build` should Open edX adopt `@tanstack/react-query` more broadly.